_gitbook/conventions/README.md

```diff
@@ -0,0 +1,3 @@
+# Conventions
+
+If you follow these conventions, your code will be more accessible to other developers. If you use a standard [coding style](coding_style.md), your project will be navigable and readable to others. If you write [documentation](documenting_code.md), others will understand the intent and behavior of your code.
```

Comment:
I suggest a formal tone in introduction. Something like:

```markdown
Code conventions serve as a guide to make code be more accessible among developers.
* [Standard coding style](coding_style.md) tends to improve readability.
* [Documentation guide](documenting_code.md) tends to improve style and format.
```

The [documenting_code.md](https://github.com/manastech/crystal/blob/gh-pages/_gitbook/documenting_code/README.md#generate-documentation) also serves to make developers easier to generate documentation with `crystal doc` command. It will be generated correctly only if it is in parser-readable format.